doc/news/changes/minor/20200128SimonSticko

```diff
@@ -0,0 +1,3 @@
+Add an optional std::function parameter to make_flux_sparsity_pattern which can
+be used to specify over which faces there should be a flux coupling in the
+created sparsity pattern.
```

Comment:
Please add your name to the end of the file, as we do in all of the other files. Also don't forget to end the file with a newline!

Reply:
Okey, done.